src/app/COMPONENTS/create-task/create-task.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup, Validators, FormBuilder } from '@angular/forms';
+import { ReactiveFormsModule } from '@angular/forms';
+import { Task } from '../../MODELS/task.model';
+import { MaterialModule } from '../../material.module';
+import { TasksService } from '../../SERVICE/tasks.service';
+
+
+
+
+@Component({
+  selector: 'app-create-task',
+  standalone: true,
+  imports: [ReactiveFormsModule, MaterialModule],
+  templateUrl: './create-task.component.html',
+  styleUrl: './create-task.component.scss'
+})
+
+export class CreateTaskComponent {
+  createTaskForm!: FormGroup
```

Comment:
don't use non null assertion (!) instead you can declare variables like this:
createTaskForm: FormGroup | null = null